src/components/VPopover/VPopover.vue

```diff
@@ -128,7 +129,16 @@ export default defineComponent({
     /**
      * the z-index to apply to the popover content
      */
-    zIndex: { type: Number, default: 999 },
+    zIndex: {
+      type: Number,
+      default: 50,
+      validator: (v) => [0, 10, 20, 30, 40, 50].includes(v),
```

Comment:
Could we add a `todo` comment to get these z index layers in a type somewhere, if we plan to use these as a z-index system across the frontend?

Reply:
I used the default tailwind values for z-index here: https://tailwindcss.com/docs/z-index
I guess I need to look into how I can import them from the Tailwind config object.

Reply:
oh, i see. i think it's fine then. if i remember correctly the config importing is something like this:

```js
import resolveConfig from 'tailwindcss/resolveConfig'
import tailwindConfig from './tailwind.config.js'

const tailwindconfig = resolveConfig(tailwindConfig)
```

Reply:
I added a todo comment. We need the array of valid values in several places (at least in VPopover and VPopoverContent), so it's a bigger change than I expected.

Reply:
cool, a standalone issue for that would be great. definitely doesn't need to happen here. LGTM!

Reply:
I made one! https://github.com/WordPress/openverse-frontend/issues/1742